src/ConcreteOperations/issubset.jl

```diff
@@ -197,8 +184,8 @@ end
 
 
 """
-    ⊆(P::AbstractPolytope{N}, S::LazySet{N}, [witness]::Bool=false
-     )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+    ⊆(P::AbstractPolytope{N}, S::LazySet{N}, witness::Bool=false;
```

Comment:
Why remove the brackets?

Reply:
It was unintentional, if you propose the change i can merge it

Reply:
```suggestion
    ⊆(P::AbstractPolytope{N}, S::LazySet{N}, [witness]::Bool=false;
```